java/server/world/entity/mob/raider.mcdoc

```diff
@@ -25,7 +29,7 @@ struct PatrolTarget {
 
 dispatch minecraft:entity[pillager] to struct Pillager {
 	...RaiderBase,
-	Inventory: [SlottedItem],
+	Inventory: [SlottedItem<255>], // TODO: Slot range is probably wrong
```

Comment:
Quick testing in latest snapshot:

- Pillagers can hold 5 items when modifying nbt directly
- They do not seem to serialize slot ids at all
- When putting some items in my inventory and copying them over to the pillager using data modify, it uses the first 5 items in the inventory array, no matter which slot ids
- This behavior is different for containers or mobs like donkeys, where it ignores any items that have slot ids which are too high.

So my guess is, for some reason this inventory does not use `SlottedItem` and just an item stack without slots?

Reply:
Thanks for testing! Fixed.